apps/spruce/src/pages/distroSettings/tabs/ProviderTab/getFormSchema.ts

```diff
@@ -242,3 +248,9 @@ export const getFormSchema = ({
     },
   },
 });
+
+const providerAccountField = {
+  type: "string" as const,
+  title: "Provider Account",
+  default: "",
+};
```

Comment:
can we put this in `schemaFields.ts` with a comment that it's only for EC2 providers (or a name it something like `ec2ProviderAccountField`)